src/components/Base.js

```diff
@@ -2,15 +2,15 @@ import Article from "../components/article/Article";
 import { CSSTransition, TransitionGroup } from "react-transition-group";
 import "../assets/css/animation.css";
 
-function Base({ setArticles, articles }) {
-  function calculateSum() {
-    let price = 0;
-    articles.forEach((article) => {
-      price += parseFloat(article.price.replace(",", ".")) * article.amount;
-    });
-    return price.toFixed(2);
-  }
+export const calculateSum = (articles) => {
```

Comment:
Not a fan of the name of the function. IMHO, it's too generic. I would not assume that I have to pass an array with an articles object in it. I would call it `calculateSumOfArticles`